src/com/company/MultiplicationMatrix.java

```diff
@@ -0,0 +1,20 @@
+package com.company;
+
+/*
+ * @param [][]matrixA - the first matrix created in the Matrix.java class
```

Comment:
Do not reference to **Matrix.java**. Your ode should be independent and can be reused by other classes.